patterns/hero-overlapped-book-cover-with-links.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Title: Hero Overlapped Book Cover with Links
+ * Slug: twentytwentyfive/hero-overlapped-book-cover-with-links
+ * Categories: banner, featured
+ * Description: A hero with an overlapped book cover and links.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:cover {"isUserOverlayColor":true,"minHeight":818,"customGradient":"linear-gradient(180deg,rgb(251,250,243) 51%,rgb(255,255,255) 51%)","contentPosition":"center center","align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast"}}},"spacing":{"padding":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|80"}}},"textColor":"contrast","layout":{"type":"constrained"}} -->
```

Comment:
Is there any reason to have a minimum height?

Reply:
It is related to Figma design. The design has 818px.